src/helpers/observer.js

```diff
@@ -36,7 +52,12 @@ export function observe(draggedEl, dropZones, intervalMs = INTERVAL_MS) {
      */
     function andNow() {
         const currentCenterOfDragged = findCenterOfElement(draggedEl);
-        const scrolled = scrollIfNeeded(currentCenterOfDragged, lastDropZoneFound);
+        const scrollableContainers = findScrollableContainers(lastDropZoneFound);
```

Comment:
do we need to find the scrollable container in every cycle? it would be much cheaper (cpu) before we call "andNow". Doable?